src/main/java/com/kaleido/fetch/service/ActivityService.java

```diff
@@ -113,40 +114,73 @@ public Activity getActivity(Long id) {
     }
 
     public ResponseEntity<String> saveActivityDraft(PlateMap plateMap) {
-    	
-    	log.info("Platemap data is ", plateMap);
-    	String plateMapURI = cabinetURI + "plate-maps";
-    	
-    	return (ResponseEntity<String>) cabinetClient.cabinetPlatemap(plateMapURI, plateMap, HttpMethod.PUT, String.class);
+        log.info("Platemap data is ", plateMap);
+        String plateMapURI = cabinetURI + "plate-maps";
+        
+        return (ResponseEntity<String>) cabinetClient.cabinetPlatemap(plateMapURI, plateMap, HttpMethod.PUT, String.class);
```

Comment:
It would probably be good to do some error handling here

Reply:
The error handling is handled on the cabinet side, where if checksum is wrong, it will send a CONFLICT response and message saying `checksum is not the most recent` back to fetch, is that enough or a different error handling is required?

Reply:
Wes, what type of error handling would want for this? Currently if there is an error in checksum values or wrong id, a CONFLICT response is sent with a message saying checksum is not the most recent from cabinet to fetch, I would prefer not to retry as checksum might get messed up.

Reply:
i was thinking more along the lines of what happens if its unreachable, etc?

Reply:
ok, will think of something that is suitable for that scenario

Reply:
I believe that is a 404 response code. I agree with the handling for the other ones

Reply:
Added a simple error handling where if response from cabinet is 404, return a response string of 404 NOT FOUND. Please check is that suitable or not